docs/sources/whatsnew/whats-new-in-v9-0.md

```diff
@@ -110,9 +110,9 @@ Refer to the [Dashboard previews topic](https://grafana.com/docs/grafana/next/da
 
 {{< figure src="/static/img/docs/dashboards/dashboard-previews.png" max-width="750px" caption="Dashboard previews" >}}
 
-## Panel title search
+## Panel title search (opt-in beta)
 
-In addition to searching dashboards by title, you can now search panels as well. If a panel’s title matches your search query, it will be displayed in the search results.
+Grafana 9 has a feature toggle `panelTitleSearch`. When enabled it tells Grafana to use a new search engine. Instead of using SQL queries, the new search uses an in-memory full-text index, and Grafana maintains this index actual upon changes in dashboards and folders. The new index should provide a better search experience, and it can search through panel titles as well as dashboard titles.
```

Comment:
```suggestion
Grafana 9 has a feature toggle `panelTitleSearch`. When enabled, it tells Grafana to use a new search engine. Instead of using SQL queries, the new search uses an in-memory full-text index, and Grafana maintains this index actual upon changes in dashboards and folders. The new index provides a better search experience and can search through panel titles as well as dashboard titles.
```

Reply:
By "actual" - do you mean current?

Reply:
@brendamuir  actual - means `up-to-date` here (i.e. index contains latest changes, has state that matches state in the database)

Reply:
then i'd say something like "Grafana keeps this index up-to-date when there are changes to dashboards and folders?

Reply:
Yep, fixed

Reply:
Done, a bit different wording – hope OK